### PR TITLE
Add another Schrader TPMS support

### PR DESCRIPTION
**Pull request by hkazu**
Schrader Electronics EG53MA4 device is a tire-pressure monitoring system (TPMS) sensor. This model comes pre-activated and is ready to be installed in compatible cars. This modification adds basic temperature and pressure readings from nearby sensors with compatible packet format.

### CODE REVIEW

**Comment by hkazu** [comment]
Pull request for issue #621. 

Updated test signals are in [https://drive.google.com/open?id=0ByPsmMWYzDTeYjZneTZSRVJRM3c](https://drive.google.com/open?id=0ByPsmMWYzDTeYjZneTZSRVJRM3c)

File: src/devices/schraeder.c [hunk 1]
```diff
@@ -43,7 +43,7 @@ static int schraeder_callback(bitbuffer_t *bitbuffer) {
 	/* Reject wrong amount of bits */
 	if ( bitbuffer->bits_per_row[0] != 68)
 		return 0;
-
+
```
Comment (zuckschwerdt) [inline src/devices/schraeder.c]: this extra space isn't needed

File: src/devices/schraeder.c [hunk 2]
```diff
@@ -84,6 +84,80 @@ static int schraeder_callback(bitbuffer_t *bitbuffer) {
...
+	uint32_t serial_id;
+	char id_str[9];
+	uint flags;
+	char flags_str[9];
+	int pressure;		// mbar
+	int temperature;	// Fahrenheit
+	int checksum;
+
+	/* Reject wrong amount of bits */
+	if ( bitbuffer->bits_per_row[0] != 120)
```
Comment (zuckschwerdt) [inline src/devices/schraeder.c]: there is extra space after the parenthesis

File: src/devices/schraeder.c [hunk 3]
```diff
@@ -84,6 +84,80 @@ static int schraeder_callback(bitbuffer_t *bitbuffer) {
...
+		fprintf(stderr, "CHECKSUM = %x\n", checksum);
+	}
+
+	data = data_make("time", "", DATA_STRING, time_str,
+					"model", "", DATA_STRING, "Schrader Electronics EG53MA4",
+					"type", "", DATA_STRING, "TPMS",
+					"flags", "", DATA_STRING, flags_str,
+ 					"id", "ID", DATA_STRING, id_str,
+					"pressure_bar",  "Pressure",    DATA_FORMAT, "%.03f bar", DATA_DOUBLE, ((double)pressure)/1000.0,
+					"temperature_C", "Temperature", DATA_FORMAT, "%.0f C", DATA_DOUBLE, fahrenheit2celsius((double)temperature),
```
Comment (zuckschwerdt) [inline src/devices/schraeder.c]: don't convert, output native native units and use the `-C si` command line switch

File: src/devices/schraeder.c [hunk 4]
```diff
@@ -84,6 +84,83 @@ static int schraeder_callback(bitbuffer_t *bitbuffer) {
...
+		fprintf(stderr, "CHECKSUM = %x\n", checksum);
+	}
+
+	data = data_make("time", "", DATA_STRING, time_str,
+					"model", "", DATA_STRING, "Schrader Electronics EG53MA4",
+					"type", "", DATA_STRING, "TPMS",
+					"flags", "", DATA_STRING, flags_str,
+ 					"id", "ID", DATA_STRING, id_str,
+					"pressure_bar",  "Pressure",    DATA_FORMAT, "%.03f bar", DATA_DOUBLE, ((double)pressure)/1000.0,
+					"temperature_F", "Temperature", DATA_FORMAT, "%.1f F", DATA_DOUBLE, (double)temperature,
```
Comment (merbanan) [inline src/devices/schraeder.c]: double is not needed, patch welcome

**Pull request merged**